stingray/io.py

```diff
@@ -714,6 +723,420 @@ def __init__(self):
         pass
 
 
+class FITSTimeseriesReader(object):
```

Comment:
Several methods in the class require docstrings

Reply:
This should be done